src/components/evm/chains.jsx

```diff
@@ -1,119 +1,164 @@
-import evm_chains from "../../data/evm_chains.json";
-import gas_services from "../../data/gas_services.json";
-import gateways from "../../data/gateways.json";
+import React, { useState, useEffect } from "react";
 import { ellipse } from "../../utils";
 import Copy from "../copy";
 import AddChain from "../web3";
 
+function toHex(decimal) {
+  return decimal.toString(16).toUpperCase();
+}
+
 export default ({ environment = "mainnet" }) => {
-  const _evm_chains = evm_chains?.[environment] || [];
-  const _gateways = gateways?.[environment] || [];
-  const _gas_services = gas_services?.[environment] || [];
+  const [chainConfigs, setChainConfigs] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        let url;
+        if (environment == "mainnet") {
+          url =
+            "https://axelar-mainnet.s3.us-east-2.amazonaws.com/configs/mainnet-config-1.x.json";
+        } else {
+          url =
+            "https://axelar-testnet.s3.us-east-2.amazonaws.com/configs/testnet-config-1.x.json";
+        }
+
+        const chainConfigResponse = await fetch(url);
+        const chainConfigData = await chainConfigResponse.json();
+        setChainConfigs(chainConfigData || []);
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      }
+    };
+
+    fetchData();
+  }, [environment]);
 
   return (
     <div className="grid lg:grid-cols-2 gap-5 not-prose">
-      {_evm_chains
-        .filter((c) => !c?.is_staging)
-        .map((c, i) => {
-          const { id, chain_id, network_id, name, provider_params, image } = {
-            ...c,
-          };
+      Items in chains:
+      {chainConfigs &&
+        chainConfigs.chains &&
+        Object.values(chainConfigs.chains).length}
+      Thanks!
```

Comment:
This appears out of place in the doc. Seems to be pushing the list of chains down 
<img width="633" alt="contract_addresses" src="https://github.com/user-attachments/assets/9097339b-a608-4f4e-91f8-7248f07920f8">